src/02-common-principles.md

```diff
@@ -378,18 +394,18 @@ for more information.
 
 ## Participant names and other labels
 
-BIDS uses custom user-defined labels in several situations (naming of
-participants, sessions, acquisition schemes, etc.) Labels are strings and MUST
-only consist of letters (lower or upper case) and/or numbers. If numbers are
-used we RECOMMEND zero padding (e.g., `01` instead of `1` if you have more than
-nine subjects) to make alphabetical sorting more intuitive.
-
-Please note that a given label is distinct from the "prefix" it refers to. For
-example `sub-01` refers to the `sub` entity (a subject) with the label `01`.
-The `sub-` prefix is not part of the subject label, but must be included in file
-names (similarly to other key names). In contrast to other labels, `run` and
-`echo` labels MUST be integers. Those labels MAY include zero padding, but this
-is NOT RECOMMENDED to maintain their uniqueness.
+BIDS allows for custom user-defined `<label>`s and `<index>`es e.g.,
+for naming of participants, sessions, acquisition schemes, etc. Note
+that they MUST consist only of allowed characters as described in
+[Definitions](02-common-principles.md#definitions) above. In `<index>`es
+we RECOMMEND using zero padding (e.g., `01` instead of `1` if you have more than
+nine subjects) to make alphabetical sorting more intuitive. Note that
+zero padding is NOT RECOMMENDED to maintain their uniqueness.
```

Comment:
```suggestion
"abusing" zero padding to maintain uniqueness of `<index>`es
is NOT RECOMMENDED.
```

Reply:
not sure about this one: although we can understand intention of using "abusing" here, I am not sure if it makes message any clearer, while introducing a loosely defined verb in quotes.

Reply:
```suggestion
zero padding SHOULD NOT be used to merely maintain uniqueness
of `<index>`es.
```

Reply:
I like it!